extensions/stackdriver/edges/edge_reporter.h

```diff
@@ -55,11 +57,12 @@ class EdgeReporter {
 
  public:
   EdgeReporter(const ::wasm::common::NodeInfo &local_node_info,
-               std::unique_ptr<MeshEdgesServiceClient> edges_client);
+               std::unique_ptr<MeshEdgesServiceClient> edges_client,
+               int batch_size = kDefaultAssertionBatchSize);
```

Comment:
nit: can we just make it a required param and pass in the default when config is not set?

Because it would just increase readability as we have another EdgeReporter constructor already.